skin/frontend/base/default/css/catalin_seo/style.css

```diff
@@ -18,4 +18,20 @@ div.price-slider .price-slider-to {
     background: url("../../images/catalin_seo/slider.png") no-repeat scroll -26px 0 transparent;
     cursor: w-resize;
     margin-top: -17px;
+}
+
+.block-layered-nav dl dd ol > li > div {
```

Comment:
That's a very specific selector. I'd suggest using a class instead. Same for the rest of the changes.